src/NexusMods.App.UI/Pages/Downloads/InProgressViewModel.cs

```diff
@@ -472,6 +477,12 @@ private void UpdateWindowInfoInternal()
         }
 
         _throughputValues.Add(new DateTimePoint(DateTime.Now, throughput));
+
+        const int delay = 5;
+        if (_throughputValues.Count > delay)
+        {
+            _lineSeries.IsVisible = _throughputValues.TakeLast(delay).Sum(x => x.Value!.Value) > 0.0;
+        }
```

Comment:
If you open the page with no downloads, the line will only disappear after 5 seconds